tests/integration/service_test.py

```diff
@@ -245,6 +246,18 @@ def test_create_container_with_volumes_from(self):
         self.assertIn(volume_container_2.id,
                       host_container.get('HostConfig.VolumesFrom'))
 
+    def test_create_container_with_volumes_from_read_only(self):
+        volume_service = self.create_service('data')
+        volume_container_1 = volume_service.create_container()
+        volume_container_2 = Container.create(self.client, image='busybox:latest', command=["/bin/sleep", "300"])
```

Comment:
Please use `top`, we changed everything over to use it so the tests would run faster